plugins/node/opentelemetry-instrumentation-express/test/express.test.ts

```diff
@@ -620,20 +620,23 @@ describe('ExpressInstrumentation', () => {
         //     `- span 'middleware - simpleMiddleware'
         //     `- span 'request handler - /post/:id'
         const spans = collector.sortedSpans;
-        assert.strictEqual(spans[0].name, 'GET /post/:id');
-        assert.strictEqual(spans[0].kind, SpanKind.CLIENT);
-        assert.strictEqual(spans[1].name, 'middleware - query');
-        assert.strictEqual(spans[1].kind, SpanKind.SERVER);
+        assert.strictEqual(spans[0].name, 'GET');
+        assert.strictEqual(spans[0].kind, testUtils.OtlpSpanKind.CLIENT);
+        assert.strictEqual(spans[1].name, 'GET /post/:id');
+        assert.strictEqual(spans[1].kind, testUtils.OtlpSpanKind.SERVER);
```

Comment:
reviewer note: this was previously asserted to be a client span, which it actually was not. Since the SDK-internal representation for a client span is `2` (`SpanKind.CLIENT`) and the OTLP represenation for a server span is `2` (`OtlpSpanKind.SERVER`), the test passed anyway.

It now asserts on the correct kind.